crates/pathfinder/src/rpc/v02/method/estimate_fee.rs

```diff
@@ -35,44 +34,8 @@ impl From<crate::cairo::ext_py::CallFailure> for EstimateFeeError {
             InvalidEntryPoint => Self::InvalidMessageSelector,
             ExecutionFailed(e) => Self::Internal(anyhow::anyhow!("Internal error: {}", e)),
             // Intentionally hide the message under Internal
```

Comment:
```suggestion
```

Reply:
Ah, I didn't actually want to commit that change... (I changed that just for debugging.)